src/physics/matter-js/components/Gravity.js

```diff
@@ -5,20 +5,20 @@
  */
 
 /**
- * [description]
+ * A component to manipulate gravity for Matter.js bodies.
  *
  * @name Phaser.Physics.Matter.Components.Gravity
  * @since 3.0.0
  */
 var Gravity = {
 
     /**
-     * [description]
+     * Ignore gravity.
      *
      * @method Phaser.Physics.Matter.Components.Gravity#setIgnoreGravity
      * @since 3.0.0
      *
-     * @param {boolean} value - [description]
+     * @param {boolean} value - Flag to ignore gravity.
```

Comment:
Should be something like:

Set to `true` to ignore the effect of world gravity, or `false` to include it.